outputs.tf

```diff
@@ -48,6 +48,11 @@ output "cluster_oidc_issuer_url" {
   value       = flatten(concat(aws_eks_cluster.this[*].identity[*].oidc.0.issuer, [""]))[0]
 }
 
+output "cluster_oidc_connect_provider_arn" {
+  description = "The ARN of the IAM OpenID Connect provider"
+  value       = flatten(concat(aws_iam_openid_connect_provider.oidc_provider[*].arn, [""]))[0]
```

Comment:
Hi. The `flatten` is unnecessary here. It was used in `cluster_oidc_issuer_url` due to the double wildcards.